pkg/devfile/versions/common/types.go

```diff
@@ -0,0 +1,210 @@
+package common
+
+// -------------- Supported devfile project types ------------ //
+// DevfileProjectsType store valid devfile project types
+type DevfileProjectsType string
```

Comment:
would `DevfileProjectType` be better type name here?

Reply:
Yes. Done!